lib/fastlane/plugin/wpmreleasetoolkit/actions/common/update_pull_requests_milestone_action.rb

```diff
@@ -0,0 +1,104 @@
+require 'fastlane/action'
+require_relative '../../helper/github_helper'
+
+module Fastlane
+  module Actions
+    class UpdatePullRequestsMilestoneAction < Action
+      def self.run(params)
+        repository = params[:repository]
+
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
+
+        pr_number = params[:pr_number]
+        from_milestone = params[:from_milestone]
+        to_milestone = params[:to_milestone]
+        pr_comment = params[:pr_comment]
+
+        target_milestone = nil
+        unless to_milestone.nil?
+          target_milestone = github_helper.get_milestone(repository, to_milestone)
+          UI.user_error!("Unable to find target milestone matching version #{to_milestone}") if target_milestone.nil?
+        end
+
+        prs_nums = if pr_number
+                     [pr_number]
+                   elsif from_milestone
+                     # get the milestone object based on title starting text
+                     m = github_helper.get_milestone(repository, from_milestone)
+                     UI.user_error!("Unable to find source milestone matching version #{from_milestone}") if m.nil?
+
+                     # get all open PRs in that milestone
+                     github_helper.get_prs_for_milestone(repository: repository, milestone: m).map(&:number)
+                   else
+                     UI.user_error!('One of `pr_number` or `from_milestone` must be provided, to indicate which PR(s) to update')
+                   end
+
+        UI.message("Updating milestone of #{prs_nums.count} PRs to `#{target_milestone&.title}`")
+
+        prs_nums.each do |pr_num|
+          github_helper.set_pr_milestone(
+            repository: repository,
+            pr_number: pr_num,
+            milestone: target_milestone
+          )
+          next if pr_comment.nil? || pr_comment.empty?
+
+          github_helper.comment_on_pr(
+            project_slug: repository,
+            pr_number: pr_num,
+            body: pr_comment,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/release-toolkit/blob/a45e842d8fb56f7fa70e4634b285dd8001a01adf/lib/fastlane/plugin/wpmreleasetoolkit/actions/common/update_pull_requests_milestone_action.rb#L49"></span>Avoid comma after the last parameter of a method call.</td>
    </tr>
  </tbody>
</table>